### ISSUE TITLE
New Version cannot be installed

### ISSUE THREAD

**Issue by Brolofmeistertv** [PhantomBot/PhantomBot#3508]
PhantomBot Version:3.13.0.0
O/S Version: Ubuntu
Java Version: openjdk version "19.0.2" 
Web Browser Version (if Panel related):

Detailed Description:
Phantombot cannot be installed

Steps to Reproduce:

Logs:
Status: Image is up to date for ghcr.io/parkervcp/yolks:java_19 
[Pterodactyl Daemon]: Finished pulling Docker container image
container@pterodactyl~ java -version
openjdk version "19.0.2" 2023-01-17
OpenJDK Runtime Environment Temurin-19.0.2+7 (build 19.0.2+7)
OpenJDK 64-Bit Server VM Temurin-19.0.2+7 (build 19.0.2+7, mixed mode, sharing)
container@pterodactyl~ java --add-opens java.base/java.lang=ALL-UNNAMED -Djava.security.policy=config/security -Dinteractive -Xms1m -Dfile.encoding=UTF-8 -jar PhantomBot.jar
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because "s" is null
        at java.base/java.lang.String.contains(String.java:2862)
        at com.gmt2001.util.LogFilter.filter(LogFilter.java:28)
        at com.gmt2001.Console.out.println(out.java:47)
        at com.gmt2001.Console.out.println(out.java:43)
        at tv.phantombot.PhantomBot.main(PhantomBot.java:1168)
container@pterodactyl~ Server marked as offline...
[Pterodactyl Daemon]: ---------- Detected server process in a crashed state! ----------
[Pterodactyl Daemon]: Exit code: 1
[Pterodactyl Daemon]: Out of memory: false
[Pterodactyl Daemon]: Aborting automatic restart, last crash occurred less than 60 seconds ago.

Screenshots (if applicable):

**Comment by GamertechAU** [comment]
Can confirm. Getting this repeated over and over via the gmt2001/phantombot-stable:latest docker image:

```
Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.lang.CharSequence.toString()" because "<parameter1>" is null

	at java.base/java.lang.String.contains(Unknown Source)

	at com.gmt2001.util.LogFilter.filter(LogFilter.java:28)

	at com.gmt2001.Console.out.println(out.java:47)

	at com.gmt2001.Console.out.println(out.java:43)

	at tv.phantombot.PhantomBot.main(PhantomBot.java:1168)

/opt/PhantomBot
```
Downgrading to the 3.12.1.0 tag worked and seems to be running correctly.

**Comment by Braastos** [comment]
This Error has been fixed with #3507 
The version with the fix is 3.13.0.1

**Comment by GamertechAU** [comment]
Can confirm, 3.13.0.1 correctly loads for me again. Thanks :)
